slurm_ops_manager/slurm_deb_manager.py

```diff
@@ -42,40 +41,51 @@ def munge_version(self) -> str:
         version = subprocess.check_output(cmd, shell=True)
         return version.decode().split(":")[-1].strip()
 
+    def _setup_ppas(self, use_stable: bool, custom_ppa: str) -> bool:
+        """Do setup a custom repository to install Slurm.
```

Comment:
The linter did allow the docstring to be `Setup a custom repository to install Slurm.` because that is not in the imperative form.